streams-contrib/streams-persist-elasticsearch/src/main/java/org/apache/streams/elasticsearch/ElasticsearchPersistWriter.java

```diff
@@ -181,11 +179,11 @@ public void cleanUp() {
             refreshIndexes();
 
             LOGGER.debug("Closed ElasticSearch Writer: Ok[{}] Failed[{}] Orphaned[{}]", this.totalOk.get(), this.totalFailed.get(), this.getTotalOutstanding());
-            timer.cancel();
 
         } catch (Throwable e) {
             // this line of code should be logically unreachable.
-            LOGGER.warn("This is unexpected: {}", e);
+            LOGGER.warn("This is unexpected: {}", e.getMessage());
+            e.printStackTrace();
```

Comment:
Not sure I understand your explanation.  Logging a throwable should print the stack trace.  If it's not in this case, I would guess that parameterizing the message string is the issue and it should be taken out.